Core/CMP API Specification.md

```diff
@@ -166,6 +160,12 @@ supportedAPIs : Array of string, // list of supported APIs (prefix strings), e.g
 
 cmpId : Number, // IAB assigned CMP ID, may be 0 during stub/loading
 
+sectionList : Array of Number, // may be empty during loading of the CMP
```

Comment:
should this say array of integer?

Reply:
since we are using "Number" elsewere, it makes sence to keep it

Reply:
I'm hoping to avoid inflating this PR any bigger than it already is.  Jan mentioned that Number is used in other parts of this document.  I'd prefer to use Number here to be consistent.  If we want to update all "Number"s that should be "Integer"s, I think that should be handled in a separate issue/pull request

Reply:
makes sense, the string specification documents say integer though